docs/ota-client-guide/modules/ROOT/pages/garage-sign-reference.adoc

```diff
@@ -203,6 +203,8 @@ Global options
 
 `root get-unsigned`: Generates an unsigned `root.json` file in a canonical JSON form.
 
+`root increment-version`: Manually increments version of unsigned `root.json`.
```

Comment:
I still think it is better to specifically point out that `root increment-version` increment manually, because there are commands like `move-offline` of `root sign` which increment version on the fly automatically in certain cases.